src/lib/core/option/option.ts

```diff
@@ -99,17 +92,22 @@ export class MdOption extends _MdOptionMixinBase implements CanDisableRipple {
   get disabled() { return (this.group && this.group.disabled) || this._disabled; }
   set disabled(value: any) { this._disabled = coerceBooleanProperty(value); }
 
+  /** Whether ripples for the option are disabled. */
+  @Input()
```

Comment:
I think that there might be some overhead to having to maintain the input. Also there's the extra JS being generated because of the getter and setter, although we can't really avoid those because it has to `markForCheck`.